src/test/codegen/function-arguments.rs

```diff
@@ -197,26 +213,36 @@ pub fn str(_: &[u8]) {
 // CHECK: @trait_borrow({{\{\}\*|ptr}} noundef nonnull align 1 %_1.0, {{.+}} noalias noundef readonly align {{.*}} dereferenceable({{.*}}) %_1.1)
 // FIXME #25759 This should also have `nocapture`
 #[no_mangle]
-pub fn trait_borrow(_: &Drop) {
+pub fn trait_borrow(_: &dyn Drop) {
+}
+
+// CHECK: @option_trait_borrow({{\{\}\*|ptr}} noundef align 1 %x.0, ptr %x.1)
```

Comment:
```suggestion
// CHECK: @option_trait_borrow({{\{\}\*|ptr}} noundef align 1 %x.0, {{i8\*|ptr}} %x.1)
```
For llvm 13 support.